Source/SpatialGDK/Public/SpatialInterop.h

```diff
@@ -15,12 +15,20 @@ class USpatialActorChannel;
 class USpatialPackageMapClient;
 class USpatialNetDriver;
 
+namespace improbable
+{
+	namespace unreal 
+	{
+		class GlobalStateManagerData;
```

Comment:
What's the reasoning behind forward declaring this?

Reply:
Returned by ```USpatialInterop::GetGlobalStateManagerData()```, and did not want to bring in more headers, since its internals are only needed in the CPP, but maybe I can change that.

Reply:
This introduces more lines of code than just including the header :)

Reply:
Added the header now